Update certificates.md

jeremiah opened the pull request:
Work towards clarification of certificate updates using clear language.

jeremiah committed:
Update certificates.md

Work towards clarification of certificate updates using clear language.

stappersg commented on docs/server/tls/certificates.md:
```diff
@@ -36,4 +36,4 @@ private-key = "%{file:/opt/stalwart-smtp/etc/private/tls.key}%"
 ## Reloading certificates
 
-When TLS certificates are updated, it is necessary to reload them in order for the changes to take effect. This can be done without stopping the server by using the [web-admin](/docs/management/webadmin/overview) or [command line interface](/docs/management/cli/overview).
+There is no automatic mechanism provided by Stalwart to automagically update the SSL/TLS certificates you're using on your domain(s). When your certificates expire, you will have to update them yourself according to this discussion on GitHub: https://github.com/stalwartlabs/stalwart/issues/463#issuecomment-2815440850. When TLS certificates are updated via the process you've chosen, it is necessar ...
```
> There is no automatic mechanism provided by Stalwart to automagically update the SSL/TLS certificates you're using on your domain(s).

But `stalwart` has  _build in  Let's Encrypt_ ...

jeremiah committed:
Update certificates.md

Provide more resources, and more accurate resources, for automated ACME updates to certs. Cleaning up some text based on feedback.

mdecimus commented:
Thanks @jeremiah. The certificates page was rewritten for v0.16 (JMAP-object configuration); the prose this PR edits no longer exists in its v0.15 form. Closing as superseded.
